src/Symfony/Component/DependencyInjection/Compiler/DecoratorServicePass.php

```diff
@@ -19,18 +19,27 @@
  *
  * @author Christophe Coevoet <stof@notk.org>
  * @author Fabien Potencier <fabien@symfony.com>
+ * @author Diego Saint Esteben <diego@saintesteben.me>
  */
 class DecoratorServicePass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
+        $definitions = new \SplPriorityQueue();
```

Comment:
@stof Can you check again? I've implemented the same solution of [ProfilerPass](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/ProfilerPass.php).